db/migrate/20220702075505_create_operations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOperations < ActiveRecord::Migration[7.0]
+  def change
+    create_table :operations do |t|
+      t.string :name, null: false
+      t.integer :amount, null: false
+      t.references :author, null: false, foreign_key: { to_table: :users }
+
+      t.timestamps
+    end
```

Comment:
- [ ] An error occurs while trying to run rails `db:migrate`. This error originates from the create_operations migrations file. Kindly fix it 🙏 
![image](https://user-images.githubusercontent.com/46839548/177015550-26b71d0a-6f33-403d-8749-1b411c0466e2.png)